inception-project/inception
# #2156 - Sidebar cluttered with unused items

**What's in the PR**
- Show external search sidebar only if there are external repos
- Show document metadata sidebar only if there is am enabled layer of type document metadata
- Show the recommender / active learning sidebars only if there are enabled recommenders
- Show image sidebar only if there is an enabled feature of type image-url on any enabeld layer
- Disable the document info sidebar - that was mainly a toy to get the sidebar off the ground

**How to test manually**
* Disable search in the settings.properties file
* Enter into a project as a plain annotator
* See that there is no sidebar anymore at all
* Fulfill any of the prerequisites for sidebar items to show up and check that the corresponding sidebars appear

**Automatic testing**

**Documentation**
